<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display_LCD</class>
 <widget class="QWidget" name="Display_LCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_air">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>261</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/设备/image/空调 (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_inside_temp">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>261</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_time">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>780</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/背景图及logo/时钟背景2);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_humi">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/设备/image/加湿器.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_inside_humi">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>440</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_outside_temp">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>260</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_outside_humi">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>440</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_equality">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/设备/image/空气清新器.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_inside_equality">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>600</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_outside_equality">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>600</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>770</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/下一页按钮.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="play_music">
   <property name="geometry">
    <rect>
     <x>1675</x>
     <y>30</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/音乐.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_music">
   <property name="geometry">
    <rect>
     <x>1675</x>
     <y>30</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/暂停音乐.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
